src/components/ScrollSpy/ScrollSpy.tsx

```diff
@@ -22,27 +22,30 @@ export interface ScrollSpyProps {
   items: SpyItem[];
   className: string;
   offset?: number;
+  scrollTarget?: Element | null;
 }
 
 interface State {
   activeSessionId: string | null;
+  scrollTarget: Element | typeof window;
 }
 
 export class ScrollSpy extends React.Component<ScrollSpyProps, State> {
   constructor(props: ScrollSpyProps) {
     super(props);
     this.state = {
       activeSessionId: null,
+      scrollTarget: props.scrollTarget ? props.scrollTarget : window,
```

Comment:
`props.scrollTarget || window`?

Reply:
yup, that's more succinct. I'll update it